tools/plugins/com.liferay.ide.upgrade.tasks.ui/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-SymbolicName: com.liferay.ide.upgrade.tasks.ui;singleton:=true
 Bundle-Vendor: Liferay, Inc.
 Bundle-Version: 3.6.0.qualifier
-Require-Bundle: com.liferay.ide.project.ui,
+Require-Bundle: com.liferay.ide.gradle.core,
```

Comment:
Hey Terry,

I don't want to add a dependency to gradle.core from tasks.ui.  If we need, we can introduce a new OSGi service.